### ISSUE TITLE
Recognize inline tables

### ISSUE THREAD

**Issue by cespare** [cespare/vim-toml#13]
https://github.com/toml-lang/toml#inline-table

**Comment by michael-younkin** [comment]
They seem to work fine for me. Did you want different syntax highlighting for the table name or something like that?

**Comment by cespare** [comment]
Yeah, it mostly works. Ideally we'd match all the various bits of syntax:

* `=` should be recognized (we don't highlight it at all today, even outside inline tables)
* `{` and `}` should be recognized as inline table braces
* `,` should be recognized as well

**Comment by cespare** [comment]
(For instance the analogous elements in JSON are all recognized.)

**Comment by dbluhm** [comment]
I've noticed this a few times with Rust's `Cargo.toml` files:

![Screenshot from 2019-03-19 22-10-21](https://user-images.githubusercontent.com/22032832/54658600-d70b4a00-4a93-11e9-81b9-1090b8eb2a6d.png)

Something about the strings and lists seems to get it.
